vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -103,18 +113,27 @@ def get_quant_method(
     ) -> Optional["QuantizeMethodBase"]:
         from vllm.attention.layer import Attention  # Avoid circular import
 
-        # Check if the layer is skipped for quantization.
-        # TODO (@robertgshaw2): support module names
-        if should_ignore_layer(prefix,
-                               ignore=self.ignore,
-                               fused_mapping=self.packed_modules_mapping):
-            return UnquantizedLinearMethod()
```

Comment:
I don't remember the significance of having this function return None vs UnquantizedLinearMethod - why is it okay to remove the ignoring here?

Reply:
`get_quant_method` only returns None if the module is not a linear, attention, or moe. If it's a linear, it will always return either UnquantizedLinearMethod or CompressedTensorsLinearMethod (theoretically).

This ignoring has been moved to within the `get_scheme` method so that quantization ignores won't affect transform matching.